egs/wsj/s5/steps/nnet3/lstm/make_configs.py

```diff
@@ -104,6 +106,9 @@ def ParseLstmDelayString(lstm_delay):
                         help="dimension of non-recurrent projection")
     parser.add_argument("--hidden-dim", type=int,
                         help="dimension of fully-connected layers")
+    parser.add_argument("--preserve-state", type=str,
+                        help="state preserving training mode. The states of the previous minibatch are preserved in the current minibatch to achieve long time sequence training",
```

Comment:
"The states of the recurrent nodes in the previous minibatch are preserved for the next minibatch."